### PR TITLE
feat: supporters section

### PR DESCRIPTION
**Pull request by im-klau**
## Jira issue

JIRA issue: [FSU-116](https://klau.atlassian.net/browse/FSU-116)

## Describe your changes
This PR introduces the Supporters section. The `help` illustration will be replaced as soon as the design team prepares the new asset.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/HomePage/components/Supporters/Supporters.styled.js [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+    #ffffff;
+
+  ${({ theme }) => theme.breakpoints.tabletLg} {
+    padding: 100px 0;
+  }
+
+  ${({ theme }) => theme.breakpoints.mobileLg} {
+    padding: 60px 0;
+  }
+`
```
Comment (nata-kab) [inline packages/frontend/app/templates/HomePage/components/Supporters/Supporters.styled.js:20]: Almost perfect 🤩 
Screenshot 2023-03-22 at 07 53 39
Screenshot 2023-03-22 at 07 53 03
As I can see, there should be no linear gradient in mobile view 😉 🕵️

File: packages/frontend/app/templates/HomePage/components/Supporters/Supporters.jsx [hunk 2]
```diff
@@ -1,24 +1,51 @@
...
-    </Styled.Grid>
-  </div>
-)
+const Supporters = ({ data }) => {
+  const theme = useTheme()
+
+  return (
+    <Styled.Wrapper>
+      <Container>
+        <Typography variant="h2">Dziękujemy za wsparcie</Typography>
```
[the remark this answers is not in the document]
Comment (im-klau) [reply packages/frontend/app/templates/HomePage/components/Supporters/Supporters.jsx]: 🙈 
The heart icon has been added ✅

**Review by AmandaKr — COMMENTED** [review]
This icon is truncated 🤔 
image

**Review by paulgrym — APPROVED** [review]
Looks really good! I fixed my mistake from actions section - I didn't replace svg width and height with viewBox so now Icon you used from that section displays properly 😁

**Comment by im-klau** [comment]
> Looks really good! I fixed my mistake from actions section - I didn't replace svg width and height with viewBox so now Icon you used from that section displays properly 😁

Ohh, thank you! I didn't noticed the problem with SVG 😅

**Review by AmandaKr — APPROVED** [review]
Everything looks great 😍

**Pull request merged**